scripts/reindex.py

```diff
@@ -143,6 +144,10 @@ def main(argv: list[str]):
     reject(args.deindex and (args.delete or args.create),
            '--deindex is incompatible with --create and --delete.')
 
+    assert azul.is_valid_failed_queues_state(), R(
+        'Operation canceled due to non-empty failed queues. Running manage_queues.py is '
+        'required to empty the queues.')
```

Comment:
```suggestion
    assert azul.is_valid_fail_queues_state(), R(
        'Operation canceled due to non-empty fail queues. Use `manage_queues.py` to empty the queues before retrying.')
```

This assert occurs in three places (two in `reindex.py` and one in `integration_test.py`). Consider adding a method `assert_fail_queues_empty()` to `azulclient.py` and replacing the asserts with a call to that.